presto-docs/src/main/sphinx/admin/properties-session.rst

```diff
@@ -2,12 +2,12 @@
 Presto Session Properties
 =========================
 
-This section describes session properties that may be used to tune 
+This section describes session properties that may be used to tune
```

Comment:
These whitespace changes are being made automatically by my editor.

"might as well" clean up these white space issues. I created a separate PR for that: [26736](https://github.com/prestodb/presto/pull/26736)